galaxy/export_user_files.py

```diff
@@ -48,10 +48,14 @@ def change_path( src ):
 
     galaxy_root_dir = os.environ.get('GALAXY_ROOT', '/galaxy-central/')
 
-    if os.path.exists( '/export/.distribution_config/' ):
-        shutil.rmtree( '/export/.distribution_config/' )
-    shutil.copytree( os.path.join(galaxy_root_dir, 'config'), '/export/.distribution_config/' )
-
+    galaxy_distrib_paths = {'/galaxy-central/config/': '/export/.distribution_config',
+                            '/galaxy-central/lib': '/export/galaxy-central/lib',
+                            '/galaxy-central/.venv': '/export/galaxy-central/.venv',
```

Comment:
@bgruening 
Thinking about it, this is probably a terrible idea (at least for the .venv), since I think this will require the cluster to run the same python version, AND be binary compatible.
But what about copying the requirements.txt and then people just need to create the .venv based on the requirements.txt ?